core/src/test/java/org/web3j/protocol/core/ResponseTest.java

```diff
@@ -1061,6 +1061,63 @@ public void testeEthGetTransactionReceiptAfterByzantium() {
                 equalTo(transactionReceipt));
     }
 
+    @Test
+    public void testTransactionReceiptIsStatusOK() {
+        TransactionReceipt transactionReceipt =
+                new TransactionReceipt(
```

Comment:
Again, why not construct a default instance and use a setter?

Reply:
Using default constructor now.